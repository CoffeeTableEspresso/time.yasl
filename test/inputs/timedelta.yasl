const time = __require_c__('time')

echo time.timedelta.frommilliseconds(2345)
echo time.timedelta.fromseconds(100)
echo time.timedelta.fromminutes(2)
echo time.timedelta.fromhours(1)
