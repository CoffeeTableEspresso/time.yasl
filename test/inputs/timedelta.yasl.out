timedelta(2.345)
timedelta(100.000)
timedelta(120.000)
timedelta(3600.000)
